index.js

```diff
@@ -6,7 +6,13 @@ try {
 }
 
 function with_query_strings (request) {
-  request._query = [Date.now().toString()]
+  var timestamp = Date.now().toString()
+  if (request._query !== undefined && request._query[0]) {
+    request._query[0] += '&buster=' + timestamp
```

Comment:
We should omit the `buster=` portion - it's possible a user will want to use `buster` themselves.